docs/content/releases.md

```diff
@@ -6,6 +6,70 @@ doctypes: ["concept"]
 toc: true
 docs: "DOCS-616"
 ---
+
+## NGINX Ingress Controller 3.0.0
+
+12 January 2023
+
+OVERVIEW:
+
+* Added support for [Deep Service Insight](https://docs.nginx.com/nginx-ingress-controller/logging-and-monitoring/service-insight) for VirtualServer and TransportServer using the [-enable-service-insight](https://docs.nginx.com/nginx-ingress-controller/configuration/global-configuration/command-line-arguments/#-enable-service-insight) cli argument.
+* *The minimum supported version of Kubernetes is now 1.21*. NGINX Ingress Controller 3.0.0 removes support for `k8s.io/v1/Endpoints` API in favor of `discovery.k8s.io/v1/EndpointSlices`. For older Kubernetes versions, use the 2.4.x release of the Ingress Controller.
+* Added support for [EndpointSlices](https://kubernetes.io/docs/concepts/services-networking/endpoint-slices/).
+* Added support to dynamically reconfigure namespace watchers using labels  [-watch-namespace-label](https://docs.nginx.com/nginx-ingress-controller/configuration/global-configuration/command-line-arguments/#-watch-namespace-label-string) and watching secrets using the [-watch-secret-namespace](https://docs.nginx.com/nginx-ingress-controller/configuration/global-configuration/command-line-arguments/#-watch-secret-namespace-string) cli arguments.
+* Allow configuration of NGINX directives `map-hash-bucket-size` and `map-hash-max-size` using the [ConfigMap resource](https://docs.nginx.com/nginx-ingress-controller/configuration/global-configuration/configmap-resource#general-customization) .
+* Added support for [fetching JWKs from a remote URL](https://docs.nginx.com/nginx-ingress-controller/configuration/policy-resource#jwt-using-jwks-from-remote-location
+) to dynamically validate JWT tokens and optimize performance through caching.
+* NGINX Service Mesh now supports the open source (free) version of NGINX Ingress Controller.
+* NGINX Ingress Controller + NGINX App Protect Denial of Service is now available through the AWS Marketplace.
+
+
+FEATURES:
+
+* [3260](https://github.com/nginxinc/kubernetes-ingress/pull/3260) Add support for EndpointSlices.
+* [3299](https://github.com/nginxinc/kubernetes-ingress/pull/3299) Support Dynamic namespaces using Labels.
+* [3261](https://github.com/nginxinc/kubernetes-ingress/pull/3261) Deep service insight endpoint for virtual server.
+* [3361](https://github.com/nginxinc/kubernetes-ingress/pull/3361) Add healthcheck for transport server.
+* [3347](https://github.com/nginxinc/kubernetes-ingress/pull/3347) Import JWKS from URL on JWT policy.
+* [3274](https://github.com/nginxinc/kubernetes-ingress/pull/3274) Allow configuration of map-hash-bucket-size and map-hash-max-size directives.
+* [3376](https://github.com/nginxinc/kubernetes-ingress/pull/3376) NGINX Service Mesh now supports the open source (free) version of NGINX Ingress Controller.
```

Comment:
NGINX Service Mesh will support the free version of NGINX Ingress Controller when using NGINX open source.